### DISCUSSION TITLE
Lualine config not working properly

### DISCUSSION THREAD

**Discussion by PCesteban** [LunarVim/LunarVim#4564] [Q&A] [answered]
I have set up some lualine custom config like this: 

```
lvim.builtin.lualine.style = "default"
lvim.builtin.lualine.sections.lualine_b = { { "branch", icon = "" }, "diff", "diagnostics" }
lvim.builtin.lualine.options.theme = "nord"
```
However some icons (Diagnostics) are not displayed properly: 
![image](https://github.com/LunarVim/LunarVim/assets/48070832/a2d755d4-fae3-4fd9-a2ee-4b24b422543f)
This does not happen with the default config: 
![image](https://github.com/LunarVim/LunarVim/assets/48070832/87822284-56dc-4c8f-863b-7afd94731d29)

Any ideas on how can I solve this? Thank you in advance.

**Comment by frolvanya** [comment]
I've commented my whole config and used code that you provided:
<img width="1512" alt="image" src="https://github.com/LunarVim/LunarVim/assets/59515280/a80492e8-d108-46ed-ba90-976b715a41c2">

Everything works just fine, so it's you local issue with fonts (but it's strange, because as I see your powerline was displayed without any issues)

**Reply by PCesteban** [reply]
Yes, I'm sure the fonts are not the issue, in your screenshot I don't see diagnostics symbols, only diff symbols, the issue is with the diagnostics, the diagnostics error symbol for example, in my first screenshot is not displayed, in my second screenshot it is displayed, so the fonts are not the issue.

**Reply by frolvanya** [reply]
Oh, sorry I didn’t understand you correctly. Then I’d check how lunarvim configures default lualine and check lvim/ui directory. I’ll check how diagnostic symbol look like on my machine in few hours

**Reply by frolvanya** [reply]
@PCesteban as you can see diagnostics error symbol is being displayed correctly.
<img width="755" alt="image" src="https://github.com/LunarVim/LunarVim/assets/59515280/3a11ad73-d89d-4fa2-bb71-3e8e6bb52457">

1) It's 100% your local issue. No issues about that in LunarVim repo + no issues in lualine.nvim + it works fine on my machine
2) Check if your behaviour persists in different terminals and in different versions (release-1.3 and release-1.4)
3) Check that all `lvim.icons.diagnostics` icons are displayed correctly (just open `~/.local/share/lunarvim/lvim/lua/lvim/icons.lua` and find diagnostics section)
Your file should look like this:
<img width="344" alt="image" src="https://github.com/LunarVim/LunarVim/assets/59515280/387647c0-7941-41d1-9f50-2b6fc7545078">

4) If they are correct, replace them with some normal ASCII letters (like X for error) just for a quick test
5) What font do you use?
6) Default works for you because it uses different icon (`BoldError` instead of `Error`)

**Reply by PCesteban** [reply, marked as answer]
So I had version 1.3, I uninstalled and reinstalled version 1.4, and now it works!

**Reply by frolvanya** [reply]
Glad to hear that! Don't forget to change discussion status to "answered", maybe someone also struggles with this strange issue

**Reply by PCesteban** [reply]
sure thing! Thank you!
